distr/README.txt

```diff
@@ -476,7 +476,7 @@ In a workspace:
         font-family: 'Fira Code';
     }
 
-4. Click Aplly or OK
+4. Click Apply or OK
```

Comment:
I haven't run this program. I believe this correction was automatically selected by Google Sheets... But, I'm fairly confident the correction is correct.